app/src/main/java/com/sdpteam/connectout/common/EntityID.java

```diff
@@ -0,0 +1,35 @@
+package com.sdpteam.connectout.common;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.Objects;
+
+public class EntityID {
+
+    protected final String value;
+
+    protected EntityID(String value) {
+        this.value = requireNonNull(value);
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        return value.equals(((EntityID) o).value);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(value);
+    }
+
+    @Override
+    public String toString() {
+        return value;
+    }
+}
```

Comment:
I wonder if this approach is the correct one, clearly using classes for the Id makes the code more intuitive, but this might be too much as `EntityID` behaves as a `String`, what do you think ?

Reply:
Following our discussion, we will open a ticket for this, since it requires multiple changes in the code and the unit tests, which makes sense to solve apart from this current task on Event UI.

Reply:
I agree, let's keep this PR focused on the demanded task. As choosing how to  manage IDs is more complexe than it seems.